instrumentation-core/spring/build.gradle

```diff
@@ -0,0 +1,36 @@
+plugins {
+	id 'org.springframework.boot' version '2.3.0.RELEASE'
+	id 'io.spring.dependency-management' version '1.0.9.RELEASE'
+	id 'java'
+}
+
+group = 'io.opentelemetry.contrib'
+version = '0.0.1-SNAPSHOT'
+sourceCompatibility = '11'
+
+configurations {
+	compileOnly {
+		extendsFrom annotationProcessor
+	}
+}
+
+repositories {
+	mavenCentral()
+}
+
+dependencies {
+	implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
+	implementation 'org.springframework.boot:spring-boot-starter-web'
+	compileOnly 'org.projectlombok:lombok'
+	annotationProcessor 'org.projectlombok:lombok'
+	testImplementation('org.springframework.boot:spring-boot-starter-test') {
+		exclude group: 'org.junit.vintage', module: 'junit-vintage-engine'
+	}
+	compile "io.opentelemetry:opentelemetry-api:0.4.0"
+	compile "io.opentelemetry:opentelemetry-sdk:0.4.0"
```

Comment:
0.5.0 is the latest version, I'd suggest using

```
apply from: "$rootDir/gradle/java.gradle"
```

at the top of the gradle file, and then you can use

```
compile deps.opentelemetryApi
compile deps.opentelemetrySdk
```